tests/Doctrine/Migrations/Tests/Metadata/Storage/TableMetadataStorageTest.php

```diff
@@ -216,9 +217,13 @@ public function testCompleteWillAlwaysCastTimeToInteger(): void
         $config     = new TableMetadataStorageConfiguration();
         $executedAt = new DateTimeImmutable('2010-01-05 10:30:21');
 
+        $getNativeConnection = method_exists($this->getSqliteConnection(), 'getNativeConnection')
+            ? 'getNativeConnection'
+            : 'getWrappedConnection';
```

Comment:
Let's avoid `$object->$methodName` calls for the sake of readability.

```suggestion
        $connection = $this->getSqliteConnection();
        $pdo = method_exists($connection, 'getNativeConnection')
            ? $connection->getNativeConnection()
            : $connection->getWrappedConnection();
```